extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local/LocalJobStore.java

```diff
@@ -123,9 +129,12 @@ public synchronized UUID findFirst(JobAuthorization.State jobState) {
     // Mimic an index lookup
     for (Entry<UUID, Map<String, Object>> job : JOB_MAP.entrySet()) {
       Map<String, Object> properties = job.getValue();
-      if (JobAuthorization.State.valueOf(
-              properties.get(PortabilityJob.AUTHORIZATION_STATE).toString())
-          == jobState) {
+      State state = State.valueOf(
+          properties.get(PortabilityJob.AUTHORIZATION_STATE).toString());
+
+      logger.debug("checking job {} in state {} to see if it matches {}",
```

Comment:
nit: message could be clarified, how about:

looking up first job in state {}: found job {} (state {})

Reply:
Done.